dbms/src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -243,7 +243,6 @@ void InterpreterSelectQuery::getAndLockStorageWithSchemaVersion(const String & d
             || (managed_storage->engineType() != ::TiDB::StorageEngine::DT
                 && managed_storage->engineType() != ::TiDB::StorageEngine::TMT))
         {
-            LOG_DEBUG(log, "{}.{} is not ManageableStorage", database_name, table_name);
```

Comment:
Running queries on the system table will trigger this logging, which is verbose but not provide any information. Remove it.